first pass at v2 product spec

slively opened the pull request:
I took some liberties here so please treat it more as a proposal ready for review and editing.

closes #48

jasepellerin commented:
So in the approval process, my understanding is that MVP doesn't want to be manually approving all of the projects. Would it be better to approve by default and send out the magic link in case the project needs to be deleted / changed? This way the site would operate autonomously and any input would be for curation rather than gatekeeping.

mckelveygreg commented:
I love the direction this is going!! Go open source community projects!! 🚀

slively commented:
> Would it be better to approve by default and send out the magic link in case the project needs to be deleted / changed? This way the site would operate autonomously and any input would be for curation rather than gatekeeping.

Not a bad idea, I'm willing to give that a shot if others feel the same way.

slively commented:
I guess that also means there's another state of `rejected` for projects we want to remove.

russell-osu commented:
If a project is rejected, how (and will) the author be notified? I assume they'll receive some sort of boilerplate notice, but will the reviewer be able to add comments or tick off a list of issues that motivated the rejection?

oliverdain commented on V2-PRODUCT-SPEC.md:
```diff
@@ -0,0 +1,111 @@
+
+#### MVP Studio
+
+MVP Studio is primarily concerned with promoting a positive meeting space and being available for a project is ready to kick off.
```
> being available for a project is ready to kick off.

Is grammer bad

slively replied:
is fixed

jasepellerin commented on V2-PRODUCT-SPEC.md:
```diff
@@ -0,0 +1,111 @@
+    - The email contains a 'magic link' to a page where they can edit their posting
+    - The posting has an expiration of 2 months after it was posted.
+        - If it should be extended we can allow them to just re-post it
+- Once approved (see MVP Studio workflow below), the posting will be publicly visible
```
Posting visible immediately

slively replied:
updated

jasepellerin commented on V2-PRODUCT-SPEC.md:
```diff
@@ -0,0 +1,111 @@
+**The primary workflow for MVP Studio is to change the status of a project**
+
+- A project can be in one of the following statuses
+    - pending
```
Don't need pending, problematic posts will be deleted.

slively replied:
updated

jasepellerin commented on V2-PRODUCT-SPEC.md:
```diff
@@ -0,0 +1,111 @@
+    - the password in the magic link is different than the one sent to the stakeholder
+- Link goes to a page showing all the details of the new project.
+- There is a drop down to change the status.
+- when the status goes from pending to approved a congratulatory email is sent to the stakeholder.
```
Can send this email immediately

slively replied:
updated

jasepellerin commented on V2-PRODUCT-SPEC.md:
```diff
@@ -0,0 +1,111 @@
+- When a new project is submitted a 'magic link' is sent to team@mvpstudio.org
+    - the password in the magic link is different than the one sent to the stakeholder
+- Link goes to a page showing all the details of the new project.
+- There is a drop down to change the status.
```
In addition, there could be a button to remove the project

jasepellerin replied:
In addition, there needs to be a button to remove the project. Could also have a field to add reasons if we want to notify the NP that their project was rejected.

jasepellerin commented on V2-PRODUCT-SPEC.md:
```diff
@@ -0,0 +1,111 @@
+- When a new project is submitted a 'magic link' is sent to team@mvpstudio.org
+    - the password in the magic link is different than the one sent to the stakeholder
+- Link goes to a page showing all the details of the new project.
+- There is a drop down to change the status.
```
In addition, need a button to remove project. Could include a field to add reasons for removal if we want to email the NP.

slively replied:
per conversation since we will do it manually no need for this yet

slively commented:
> If a project is rejected, how (and will) the author be notified? I assume they'll receive some sort of boilerplate notice, but will the reviewer be able to add comments or tick off a list of issues that motivated the rejection?

Added a note about how if we want to reach out we will do it manually.

> I think the only thing that's missing is the bit where we get contacted by the project owner when they think they're ready to go and we help them get started.

Added notes that this should be part of the copy we provide before they submit a project on how they would kick off, and how they can contact us at any point.
